packages/teams-js/test/public/dialog.spec.ts

```diff
@@ -52,7 +52,7 @@ describe('Dialog', () => {
         if (allowedContexts.some((allowedContext) => allowedContext === context)) {
           it(`FRAMELESS: should throw error when dialog is not supported in ${context} context`, async () => {
             await utils.initializeWithContext(context);
-            utils.setRuntimeConfig({ apiVersion: 2, supports: {} });
+            utils.setRuntimeConfig({ apiVersion: 4, supports: {} });
```

Comment:
It is not necessary to update all, but why not update to the latest?

Reply:
I'm totally fine updating to the latest -- the direction I was going was that if you want to set the precedent that latest is best here, should it just be referencing the `latestRuntimeApiVersion` constant instead of having a literal?

Reply:
good point, I will update it to the latestRuntimeApiVersion